pycargoebuild/cargo.py

```diff
@@ -299,6 +302,7 @@ def get_package_metadata(f: typing.BinaryIO,
         name=pkg_meta["name"],
         version=pkg_version,
         license=pkg_license,
+        features=pkg_features,
```

Comment:
```suggestion
        features=cargo_toml.get("features", {}),
```

Helper variables are only used when we actually need to do something with them.